src/abstractdataframe/io.jl

```diff
@@ -215,12 +273,62 @@ function Base.show(io::IO, ::MIME"text/latex", df::AbstractDataFrame)
     write(io, "\\end{tabular}\n")
 end
 
+function Base.show(io::IO, mime::MIME"text/latex", dfr::DataFrameRow)
+    r, c = parentindices(dfr)
+    _show(io, mime, view(parent(dfr), [r], c), rowid=r)
+end
+
+function Base.show(io::IO, mime::MIME"text/latex", gd::GroupedDataFrame)
```

Comment:
Maybe it would be possible to share code with other methods instead of repeating it? A small helper function taking `mime` could do the trick (for HTML, and maybe also the default output).

Reply:
Unfortunately they do not have exactly the same signature (for `"text/html"` we additionally allow a keyword argument).

I shared `DataFrameRow` mime for `"text/csv"` and `"text/tab-separated-values"` but it does not save that much.

Reply:
Actually I have to keep separate methods as I get method ambiguity with methods defined in DelimitedFiles.jl (who would have thought).

Reply:
But you could still call an internal function with shared code, right?

Reply:
The code is 90% similar, but 10% different - and the differences are scattered. I could try this, but it would contain if-else in many places as html and LaTeX control sequences have different semantics (I guess this is also the reason why original `show` functions for `AbstractDataFrame` although very similar are separate).

If you really insist I can do it, but I am not convinced that it is worth it.